### ISSUE TITLE
cloud-demo failing with CP 6.0

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/kafka-docker-playground#106]
```
18:45:00 ccloud api-key create --service-account 116195 --resource lkc-6kv2j
3779It may take a couple of minutes for the API key to be ready.
3780Save the API key and secret. The secret is not retrievable later.
378118:45:02 Wait 90 seconds for the user and service account key and secret to propagate
378218:46:32 Create a local configuration file /tmp/client.config for the client to connect to Confluent Cloud with the newly created API key and secret
378318:46:32 Write properties to /tmp/client.config:
378418:46:32 Check if topic demo-acl-topic exists
378518:46:37 Topic demo-acl-topic already exists
378618:46:37 By default, no ACLs are configured
378718:46:37 ccloud kafka acl list --service-account 116195
3788  ServiceAccountId | Permission | Operation | Resource | Name | Type  
3789+------------------+------------+-----------+----------+------+------+
379018:46:43 Run the Java producer to demo-acl-topic: before ACLs
3791
3792
3793No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
3794Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
3795
3796The build has been terminated
```

**Comment by vdesabou** [comment]
```
[2020-09-30 12:44:48,798] ERROR Uncaught exception in thread 'kafka-producer-network-thread | producer-1': (org.apache.kafka.common.utils.KafkaThread)
java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/JsonNode
        at org.apache.kafka.common.requests.ApiVersionsRequest$Builder.<clinit>(ApiVersionsRequest.java:36)
        at org.apache.kafka.clients.NetworkClient.handleConnections(NetworkClient.java:910)
        at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:555)
        at org.apache.kafka.clients.producer.internals.Sender.runOnce(Sender.java:325)
        at org.apache.kafka.clients.producer.internals.Sender.run(Sender.java:240)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.JsonNode
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        ... 6 more
```

**Comment by vdesabou** [comment]
> this was fixed shortly after 2.6.0 was released
